src/main/java/ru/sbtqa/tag/pagefactory/Page.java

```diff
@@ -808,6 +808,11 @@ public String getElementTitle(WebElement element) {
         for (Map.Entry<Field, String> entry : PageFactory.getPageRepository().get(this.getClass()).entrySet()) {
             try {
                 if (Core.getElementByField(this, entry.getKey()) == element) {
+                    if (entry.getKey().getAnnotation(ElementTitle.class) != null
```

Comment:
too complicated.
extract entry.getKey().getAnnotation(ElementTitle.class) to variable and use it.